### PR TITLE
Tzg

### PR DESCRIPTION
**Pull request by DreamerLark**
- #196 

recommit post

### CODE REVIEW

File: docs/_posts/2021-12-08-tongzhanggui-customer-platform.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+死不掉也活不好，谁能改变中国100万快消二批的命运？
```
Comment (huan) [inline docs/_posts/2021-12-08-tongzhanggui-customer-platform.md:1]: Add a YAML front matter at the beginning of the markdown file like others.

File: docs/_posts/2021-12-08-tongzhanggui-customer-platform.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+播报文章
+
+![pic](/assets/2021/12-08-tongzhanggui-customer-platform/46d4b0230deffe6a686efef8daa60089.gif)
+
+作者：陈思廷
+
+![pic](/assets/2021/12-08-tongzhanggui-customer-platform/aab533d22346508474a1b7c956cd35c4.png)
+
+**二批，快消厂家无不欲扁平之而后快**
```
Comment (huan) [inline docs/_posts/2021-12-08-tongzhanggui-customer-platform.md:11]: Use header 2 for titles (##)

**Review by huan — CHANGES_REQUESTED** [review]
Thanks for posting this great news! Please follow my reviews, and rename `tong-liqun` to `liqun-tong` to follow our naming convention.

File: docs/_posts/2021-12-08-tongzhanggui-customer-platform.md (hunk 2 above)
Comment (DreamerLark) [reply docs/_posts/2021-12-08-tongzhanggui-customer-platform.md]: fix it

> Thanks for posting this great news! Please follow my reviews, and rename `tong-liqun` to `liqun-tong` to follow our naming convention.

liqun-tong has exists。 i delete the md

**Review by huan — CHANGES_REQUESTED** [review]
Thanks for the update!

1. Please convert all `.png` to `.webp`(recommended) or `.jpg` so that we can have smaller file size. (`BIN +1.11 MB .../2021/12-08-tongzhanggui-....png `)
2. Change `author: 陈思廷` to `author: liqun-tong` so that this news can display under http://pre-angel.com/peoples/liqun-tong/ . (the original link & author can be appended to the end of this article)
3. Please sign the CLA before we can merge this article.

**Comment by huan** [comment]
One more thing: it would be great to have a meaningful filename instead of `d09540642f89918f22ac21a3a43cb3fa.png`. Thanks!

File: docs/assets/2021/12-08-tongzhanggui-customer-platform/404.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (huan) [inline docs/assets/2021/12-08-tongzhanggui-customer-platform/404.html:1]: I don't think we need this file?

**Review by huan — COMMENTED** [review]
The `BIN +545 KB .../2021/12-08-tongzhanggui-customer-platform/46d4b0230deffe6a686efef8daa60089.gif` is unnecessary huge and I believe it should be converted to `.webp` or `.jpg`, or just remove it because it's a ad banner.

**Pull request closed without merging**